lib/ardb/migration.rb

```diff
@@ -4,16 +4,18 @@ module Ardb
 
   class Migration
 
-    attr_reader :identifier, :class_name, :file_name, :file_path
-    attr_reader :source
+    attr_reader :identifier, :migrations_path
+    attr_reader :class_name, :file_name, :file_path, :source
 
-    def initialize(identifier)
+    def initialize(identifier, ardb_config)
```

Comment:
@jcredding can we switch the arg order here and pass the config instance first instead?  No great reason - it just *feels* more appropriate.  I'm not super passionate though so if you disagree just ignore this comment :).